install/helm/agones/templates/crds/fleetautoscaler.yaml

```
@@ -93,6 +93,8 @@ spec:
                   type: boolean
                 scalingLimited:
                   type: boolean
+                lastAppliedPolicy:
+                  type: string
```

Per the CRD policy, this field either needs to be optional, or have a default value. In this case it looks like the controller needs to have this field, so there should be a default value. https://github.com/googleforgames/agones/edit/main/install/helm/agones/README.md has the details.

```suggestion
                  type: string
                  default: ""
```

---

Updated now!